### PR TITLE
Ports: [READY] Bluespace entropy (#5257)

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

Ports PRs 5257, 5806, 5820 from upstream

## Why It's Good For The Game

Getting us back to date

### CODE REVIEW

**Review by Michiyamenotehifunana — APPROVED** [review]
Looks good, doesn't seem to cause any conflicts with the initialize(mapload) stuff from previous cherry picks.

File: cev_eris.dme [hunk 1]
```diff
@@ -2334,7 +2338,6 @@
 #include "code\modules\projectiles\guns\projectile\automatic\lmg.dm"
 #include "code\modules\projectiles\guns\projectile\automatic\maxim.dm"
 #include "code\modules\projectiles\guns\projectile\automatic\molly.dm"
-#include "code\modules\projectiles\guns\projectile\automatic\slaught_o_matic.dm"
```
Comment (Michiyamenotehifunana) [inline cev_eris.dme:2337]: What

Comment (Jamini) [reply cev_eris.dme:2337]: Probably got picked off while I was swapping branches. Easy to fix.

**Comment by Jamini** [comment]
Problematic line removed

**Review by Michiyamenotehifunana — APPROVED** [review]
Re-approved.

**Pull request merged**